src/Examples/Profile/47Example/47Example.csproj

```diff
@@ -23,6 +24,8 @@
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
Could this be self closing? `<NuGetPackageImportStamp />`

Reply:
I'm sure it could be, but it was added by Visual Studio automatically, so I'm inclined to leave it as it is, so it's less likely to be changed again.